operations/rack_resiliency/Enable_Rack_Resiliency_on_a_Running_System.md

```diff
@@ -0,0 +1,293 @@
+    ```
+
+1. (`ncn-m#`) Check if the rack resiliency playbook is present in the configuration
+
+    ```bash
+    cat ${CONFIG}.json | grep rack_resiliency_for_mgmt_nodes.yml
+    ```
+
+    Example Output:
+
+    ```text
+    "playbook": "rack_resiliency_for_mgmt_nodes.yml"
+    ```
+
+    **NOTE:** If the above command returns the output shown above, then
+    skip to the next step (Perform the component update). Otherwise, perform
+    the following sub-steps to add the rack resiliency layer.
+
+    1. (`ncn-m#`) Get the corresponding `csm-config` branch (@VCS) from the product catalog for CSM 1.7.0 version.
+
+        ```bash
+        kubectl get cm -n services cray-product-catalog -o yaml | yq - r 'data.csm' | grep ^1.7.0 -A 10
+        ```
+
+        Example output:
+
+        ```yaml
+        1.7.0:
+          configuration:
+            clone_url: https://vcs.cmn.odin.hpc.amslabs.hpecorp.net/vcs/cray/csm-config-management.git
+            commit: 9c30b68a29878996761f3c8280d6e57dfb79e8c8
+            import_branch: cray/csm/1.43.0
+            import_date: 2025-07-28 15:51:40.054187
+            ssh_url: git@vcs.cmn.odin.hpc.amslabs.hpecorp.net:cray/csm-config-management.git
+        ```
+
+    1. Edit `${CONFIG}.json` by adding a new layer with commit ID found in the previous step.
+
+        Add the following layer in the `{CONFIG}.json` file. Be sure to substitute the actual
+        commit ID found in the previous step. Leave all other fields as they are shown here.
+
+        ```json
+        {
+          "layers": [
+            {
+              "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
+              "commit": "commit_id_goes_here",
+              "name": "csm-ncn-rack-resiliency",
+              "playbook": "rack_resiliency_for_mgmt_nodes.yml"
+            },
+          ]
+        }
+        ```
+
+    1. (`ncn-m#`) Update the configuration in CFS.
+
+        ```bash
+        cray cfs configurations update --file ${CONFIG}.json ${CONFIG}
+        ```
+
+1. (`ncn-m#`) Perform the component update.
+
+```bash
+cray cfs components update $XNAME --state []
+```
+
+1. (`ncn-m#`) Wait for the component to be configured.
+
+    Wait until the configuration status of the component changes to `configured`.
+
+    ```bash
+    cray cfs components describe $XNAME
+    ```
+
+    In case of configuration failure, see
+    [Troubleshoot CFS Issues](../configuration_management/Troubleshoot_CFS_Issues.md).
+
+## 3. Deploy Helm chart
+
+Deploy [`cray-rrs`](cray-rrs_Deployment.md) in `rack-resiliency` namespace using the following procedure:
+
+1. (`ncn-mw#`) Get current CSM installation location.
+
+    ```bash
+    . /etc/cray/upgrade/csm/myenv
+    echo $CSM_ARTI_DIR
+    ```
+
+    Example output:
+
+    ```text
+    /etc/cray/upgrade/csm/test-activity/2148565/media/csm-1.7.0-rc.2
+    ```
+
+1. (`ncn-mw#`) Install the helm chart on the cluster.
```

Comment:
This says install, but as previously noted, it is already going to be installed.